apps/dashboard/src/components/product-pages/common/ProductButton.tsx

```diff
@@ -11,22 +10,18 @@ interface GeneralCtaProps extends Omit<LinkButtonProps, "href"> {
   title: string;
   href: string;
   size?: ButtonProps["size"];
-  iconColor?: string;
 }
 
 export const ProductButton: React.FC<GeneralCtaProps> = ({
   title,
   href,
-  iconColor = "yellow.400",
   ...props
 }) => {
   const trackEvent = useTrack();
 
   return (
     <LinkButton
-      leftIcon={
-        <Icon as={BsFillLightningChargeFill} color={iconColor} boxSize={4} />
-      }
+      leftIcon={<ZapIcon className="size-4 fill-black" />}
```

Comment:
this loses the icon color no? were we just not using it?

Reply:
We weren't using it. The icon adapts to black/white depends on the dark & light mode